components/TechStack/index.tsx

```diff
@@ -0,0 +1,158 @@
+"use client";
+import { TechStackType } from "@/types/techstack";
+import Image from "next/image";
+import SectionTitle from "../Common/SectionTitle";
+
+
+const TechStackData: TechStackType[] = [
```

Comment:
Inside the TechStack folder, create a new file called `constants.ts`, put this variable there in the `constants.ts` file, then import it to this current file.